develop/dev-guide-delete-data.md

```diff
@@ -290,4 +291,42 @@ In each iteration, `DELETE` deletes up to 1000 rows from `2022-04-15 00:00:00` t
 
 </div>
 
-</SimpleTab>
\ No newline at end of file
+</SimpleTab>
+
+## Non-Transactional Bulk-delete
+
+> **Note:**
+>
+> TiDB supports the [Non-Transactional DML Statements](/non-transactional-dml.md) feature starting with `v6.1.0`. This feature is not available on lower versions.
+
+### Prerequisites of Non-Transactional Bulk-delete
+
+Please ***carefully*** read the [Non-Transactional DML Statements](/non-transactional-dml.md) first. It enhances the performance and ease of use in batch data processing at the expense of transactional atomicity and isolation.
+
+Therefore, extreme care is required during use. Otherwise, the non-transactional feature of the operation can lead to serious consequences (e.g., data loss) in case of mishandling.
```

Comment:
```suggestion
Therefore, you should use it carefully. Otherwise, the non-transactional feature of the operation might cause serious consequences (such as data loss) due to mishandling.
```
[Google Developer Style: Don't use `e.g.`, use `for example` or `such as` instead](https://developers.google.com/style/word-list?hl=en#:~:text=to%20EAD.-,e.g.,.%20Too%20many%20people%20mix%20up%20e.g.%20and%20i.e.,-egress)